ci/docker/release/build.Dockerfile

```diff
@@ -100,41 +102,42 @@ RUN wget -q https://github.com/${HPX_FORK}/hpx/archive/${HPX_VERSION}.tar.gz -O
     rm -rf /root/hpx.tar.gz /root/hpx-${HPX_VERSION}
 
 # Install BLASPP
-ARG BLASPP_VERSION=c090b5738c8e
+ARG BLASPP_VERSION=2020.10.02
 ARG BLASPP_PATH=/usr/local/blaspp
 RUN source /opt/intel/compilers_and_libraries/linux/mkl/bin/mklvars.sh intel64 && \
-    wget -q https://bitbucket.org/icl/blaspp/get/${BLASPP_VERSION}.tar.gz -O blaspp.tar.gz && \
+    wget -q https://bitbucket.org/icl/blaspp/downloads/blaspp-${BLASPP_VERSION}.tar.gz -O blaspp.tar.gz && \
     tar -xzf blaspp.tar.gz && \
-    cd icl-blaspp-${BLASPP_VERSION} && \
+    cd blaspp-${BLASPP_VERSION} && \
     mkdir build && \
     cd build && \
     cmake .. \
-      -DBLASPP_BUILD_TESTS=OFF \
+      -DBUILD_TESTS=OFF \
       -DCMAKE_BUILD_TYPE=RelWithDebInfo \
       -DUSE_OPENMP=OFF \
+      -DUSE_CUDA=OFF \
       -DBLAS_LIBRARY='Intel MKL' \
       -DBLAS_LIBRARY_THREADING=sequential \
       -DCMAKE_INSTALL_PREFIX=$BLASPP_PATH && \
```

Comment:
Please note that blaspp cmake has been messed up... The new config should be:
```
-Dbuild_tests=OFF
-DCMAKE_BUILD_TYPE=RelWithDebInfo
-Duse_openmp=OFF
-Dblas='Intel MKL'
-Dblas_threaded=OFF
``` 
You can see from the kubernetes output too:
```
#15 10.61 CMake Warning:
#15 10.61   Manually-specified variables were not used by the project:
#15 10.61 
#15 10.61     BLAS_LIBRARY
#15 10.61     BLAS_LIBRARY_THREADING
#15 10.61     BUILD_TESTS
#15 10.61     USE_CUDA
#15 10.61     USE_OPENMP
```